libr/include/r_anal.h

```diff
@@ -576,15 +576,12 @@ typedef struct r_anal_switch_obj_t {
 	RList *cases;
 } RAnalSwitchOp;
 
-#define RANAL void*
-//struct r_anal_t*
-#define RANAL_BLOCK void*
-//struct r_anal_bb_t*
+struct r_anal_bb_t;
 typedef struct r_anal_callbacks_t {
-	int (*on_fcn_new) (RANAL, void *user, RAnalFunction *fcn);
-	int (*on_fcn_delete) (RANAL , void *user, RAnalFunction *fcn);
-	int (*on_fcn_rename) (RANAL, void *user, RAnalFunction *fcn, const char *oldname);
-	int (*on_fcn_bb_new) (RANAL, void *user, RAnalFunction *fcn, RANAL_BLOCK bb);
+	int (*on_fcn_new) (void *ana, void *user, RAnalFunction *fcn);
```

Comment:
Why not `void *anal`? Better to be consistent across the program.

Reply:
Improve the naming BTW.

Reply:
I think we don't want to follow IDA and prefer to have `anal`.

Reply:
Also, why isn't this using `RAnal`/`struct r_anal_t` type instead of `void *`?